manifests/init.pp

```diff
@@ -8,23 +8,35 @@
 class misp (
   # MISP installation
   # # MISP repositories
-  $misp_git_repo = 'https://github.com/MISP/MISP.git',
-  $misp_git_tag = 'v2.4.71',
-  $stix_git_repo = 'https://github.com/STIXProject/python-stix.git',
-  $stix_git_tag = 'v1.2.0.6',
-  $cybox_git_repo = 'https://github.com/CybOXProject/python-cybox.git',
-  $cybox_git_tag = 'v2.1.0.17',
-  $mixbox_git_repo = 'https://github.com/CybOXProject/mixbox.git',
-  $mixbox_git_tag = 'v1.0.3',
-  $maec_git_repo = 'https://github.com/MAECProject/python-maec.git',
-  $maec_git_tag = 'v4.1.0.14',
-  $pydeep_git_repo = 'https://github.com/kbandla/pydeep.git',
-  $pydeep_git_tag = 'e4ce348566293475016ca7fa9fb7fc4f61f1997f', # Using SHA from latest commit in 0.2 tag, because there is also a 0.2 branch
+  String $misp_git_repo = 'https://github.com/MISP/MISP.git',
```

Comment:
thanks for the datatypes! Are you interested in making them more strict? You can use `Stdlib::HTTPSUrl` for the urls.

Reply:
Well, I've got more datatypes and stuff coming, but I wanted to do that in a separate PR, since it's easily just as big as this one.  
In regards to `Stdlib::HTTPSUrl`, I was looking at doing just that, but at the same time I really wanted to keep the ability to use SSH git URLs. So I decided to just go with a plain string for now.

Reply:
sounds good!